tests/custom_lookups/models.py

```diff
@@ -17,3 +17,8 @@ class Article(models.Model):
 
 class MySQLUnixTimestamp(models.Model):
     timestamp = models.PositiveIntegerField()
+
+
+class Company(models.Model):
+    name = models.CharField(max_length=20)
+    place = models.CharField(max_length=20)
```

Comment:
We can reuse `Author` model. If we need two `CharField`s then we can add e.g. `alias` to the `Author`.

Reply:
Done👍